es-ds-docs/pages/organisms/cta-card.vue

```diff
@@ -0,0 +1,212 @@
+                    md="6"
+                    lg="5"
+                    xl="5"
+                    xxl="4">
+                    <es-cta-card
+                        image-first
+                        horizontal-padding="px-200"
+                        vertical-spacing="mb-50">
+                        <template #image>
+                            <nuxt-img
+                                alt="Community solar"
+                                class="mb-50 w-100"
+                                height="150"
+                                src="https://a-us.storyblok.com/f/1006156/159x150/acd049dd6f/community-solar-icon.svg"
+                                width="150" />
+                        </template>
+                        <template #heading> Community solar </template>
+                        <template #subtitle>
+                            <p>Go solar without installing panels on your roof</p>
+                        </template>
+                        <template #cta>
+                            <nuxt-link href="#">
+                                Learn about community solar
+                                <icon-arrow-right style="position: relative; top: -0.1em" />
+                            </nuxt-link>
+                        </template>
+                    </es-cta-card>
+                </b-col>
+            </b-row>
+        </div>
+
+        <div class="mb-500">
+            <h2 class="mb-200">No image with zip code form</h2>
+            <b-row class="justify-content-center justify-content-lg-start">
+                <b-col
+                    sm="8"
+                    md="6"
+                    lg="4"
+                    xl="5"
+                    xxl="4">
+                    <es-cta-card>
+                        <template #heading> Easily find what solar costs in your area </template>
+                        <template #cta>
+                            <es-zip-code-form
+                                input-id="no-image-with-zip-form"
+                                privacy-policy-link="https://www.energysage.com/privacy-policy/"
+                                url="https://www.energysage.com/market/start/">
+                                <template #buttonText> See local offers </template>
+                            </es-zip-code-form>
+                        </template>
+                    </es-cta-card>
+                </b-col>
+            </b-row>
+        </div>
+
+        <div class="mb-500">
+            <h2>EsCtaCard slots</h2>
+            <ds-prop-table
+                :rows="slotTableRows"
+                :widths="tableWidths" />
+        </div>
+
+        <div class="mb-500">
+            <h2>EsCtaCard props</h2>
+            <ds-prop-table
+                :rows="propTableRows"
+                :widths="tableWidths" />
+        </div>
+
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsCtaCard.vue"
```

Comment:
While this might work, we've updated these `comp-source` and `doc-source` values for consistency to             `comp-source="es-ds-components/components/<component>.vue"` and `doc-source="es-ds-docs/pages/molecules/<component>.vue"`
so this would be 
```
 <ds-doc-source
            :comp-code="compCode"
            comp-source="es-ds-components/components/es-cta-card.vue"
            :doc-code="docCode"
            doc-source="es-ds-docs/pages/molecules/cta-card.vue" />
```

Reply:
Missed this yeah. Fixed.